conf/locale/es_419/LC_MESSAGES/django.po

```diff
@@ -6679,8 +6679,7 @@ msgstr ""
 "Sin embargo, si cumples con tener la calificación necesaria para aprobar el curso y "
 "realizas el pago con cualquiera de los medios que ofrecemos, podrás obtener tu "
 "certificado y respaldar lo aprendido. Te alentamos a revisar todo el material, rendir"
-"tus prácticas y examen final.Si necesitas información adicional escríbenos a"
-"contacto@campusromero.pe o llámanos al +51 948430337"
+"tus prácticas y examen final."
```

Comment:
Can you please indicate how you run the compile messages command?

Reply:
django-admin.py compilemessages -v0 -l es_419